### PR TITLE
Add exactFilter option to collectLabelElements API for tiger backend

### PR DESCRIPTION
**Pull request by Vojtasii**
The in-development Metric Editor requires usage of the recently implemented exactFilter option for collectLabelElements API of Tiger backend. This PR implements the required changes.

- regenerate tiger OpenAPI client
- rename interfaces where required, resolve conflict with ActionsApi* interfaces between metadata and afm-exec
- add 'exactFilter' option to collectLabelElements in sdk-backend-tiger - mapped to the 'uris' option in sdk-backend-spi

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5179/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6885/ : FAILURE in 7m 01s

File: libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/index.ts [hunk 1]
```diff
@@ -89,18 +89,18 @@ class TigerWorkspaceElementsQuery implements IElementsQuery {
...
                 ...(options?.filter && { patternFilter: options.filter }),
+                ...(options?.uris && { exactFilter: options.uris }),
                 ...(options?.order && { sortOrder: options.order === "asc" ? "ASC" : "DESC" }),
                 limit,
                 offset,
                 label: ref.identifier,
                 workspaceId: this.workspace,
             };
-
+            console.log(elementsRequest, "el request");
```
Comment (Vojtasii) [inline libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/index.ts:103]: leftover console.log

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5182/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6888/ : SUCCESS in 15m 10s

**Comment by kandl** [comment]
extended check sonar

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5183/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6889/ : SUCCESS in 17m 48s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR1742

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5184/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/544/ : SUCCESS in 15m 01s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1742

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1481/ : SUCCESS in 1m 03s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1758/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/545/ : SUCCESS in 13m 39s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6890/ : SUCCESS in 14m 40s

**Pull request merged**